source/routing/raptor_api.cpp

```diff
@@ -1152,7 +1152,7 @@ void make_isochrone(navitia::PbCreator& pb_creator,
     auto departures = get_stop_points(origin, raptor.data, worker);
 
     if(departures.size() == 0){
-        pb_creator.set_response_type(pbnavitia::NO_ORIGIN_POINT);
+        pb_creator.fill_pb_error(pbnavitia::Error::unknown_object, "The entry point: " + origin.uri + " is not valid");
```

Comment:
hum I'm not convinced this is the right place to change, for me you should change the `get_stop_points` message no ?

it's not the same to start from a far away location or from a VJ